docs/analyst_guide.md

```diff
@@ -0,0 +1,217 @@
+
+```
+SELECT avg(x.num_individuals)
+FROM (
+  SELECT city, count(*) as num_individuals
+  FROM customers
+  GROUP BY city
+) x
+```
+
+Projections of grouping expressions in `SELECT` and the `HAVING` clause of anonymizing
+queries are also considered post processing and are accepted.
+
+**Example:** The following query post processes the grouping labels and filters the output bins.
+
+```
+SELECT 'City: ' || upper(city), count(*)
+FROM customers
+GROUP BY city
+HAVING count(*) > 10
+```
+
+If `pg_diffix.treat_unmarked_tables_as_public` is set to `false`. then tables not marked as `personal` or `public` are not queryable in any way.
+Use [diffix.show_labels()](#diffixshowlabels) to see which tables are marked.
+
+# Utility statements
+
+## Other SQL statements
+
+Most utility statements are disallowed in anonymized access level.
+Allowed SQL statements include:
+
+- `EXPLAIN` - explain is partially allowed. Options `COSTS` and `ANALYZE` are rejected.
+- `SET` - a subset of parameters can be changed by non-superusers, even in anonymized mode.
+  Note that most `pg_diffix` parameters require superuser access.
+
+Any statement that alters the database (`INSERT`, `DELETE`, `UPDATE`, `DROP`, ...) or causes any other persistent state change is forbidden.
+
+`COPY` statements and other indirect methods of reading data from the database are forbidden.
+
+## diffix.show_labels()
+
+Run `SELECT * FROM diffix.show_labels()` to see the complete list of labels on database objects.
+
+## diffix.access_level()
+
+Run `SELECT diffix.access_level()` to see the current [access level](#access-levels) of the session.
+Possible values are `direct`, `anonymized_trusted`, and `anonymized_untrusted`.
+
+# Suppress bin
+
+When performing grouping in queries that target personal tables, bins that pertain to too few individuals are suppressed.
+
+Diffix combines the counts of all suppressed rows to a special **suppress bin**.
+The suppress bin is the first row of the query result. The suppress bin may itself be suppressed.
+All grouping columns of the suppress bin are set to `*` for text-typed columns and `NULL` for other types.
+
+The configuration parameter `pg_diffix.compute_suppress_bin` controls whether the suppress bin is computed and emitted.
+
+The configuration parameter `pg_diffix.text_label_for_suppress_bin` specifies the value to use for text-typed grouping labels.
+Default value is `*`.
+
+The aggregate `diffix.is_suppress_bin(*)` returns `true` for the suppress bin, and false for any other bin.
+This can be used to identify the suppress bin if `NULL` values are ambiguous for a grouping label.
+
+# Supported functions
+
+## Count
+
+The following versions of the count aggregate are supported:
+
+- `count(*)` - count all rows.
+- `count(col)` - counts non-null occurrences of the given column.
+- `count(distinct col)` - counts distinct values of the given column.
+- `count(distinct aid)` - counts number of individuals in the bin.
```

Comment:
I think it is not necessary to list `count(distinct aid)`. It is a subset of `count(distinct col)`